### PR TITLE
Add Extract Indicators from QR Codes

### PR DESCRIPTION
**Pull request by rundssoar**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Add Indicator Extraction from QRCodes using the ReadQRCode Script

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by rundssoar** [comment]
@merit-maita Hello,
this is the resubmit of PR #33069, after trying to rebase the branch messed up the Git workflow.
I have addressed the Comments from the other PR: I updated the Image and regenerated the Docs

**Comment by merit-maita** [comment]
@rundssoar please set up a demo with @melamedbn and me.
thank you!

File: Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml [hunk 1]
```diff
@@ -1757,6 +1758,130 @@ tasks:
...
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "26":
+    id: "26"
+    taskid: eefdeebf-4065-4f99-8511-c7da64e5a6fe
+    type: regular
```
Comment (merit-maita) [inline Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml:1815]:
```suggestion
    type: regular
    skipunavailable: true
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml [hunk 2]
```diff
@@ -1757,6 +1758,131 @@ tasks:
...
+    skipunavailable: true
```
Comment (merit-maita) [inline Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml:1816]:
```suggestion
```

**Review by merit-maita — APPROVED** [review]

**Comment by merit-maita** [comment]
@rundssoar please handle the failing [job](https://app.circleci.com/pipelines/github/demisto/content/348799/workflows/dffb1779-de05-4555-9ad6-1f11765721da/jobs/742452), this could be handled by adding skipunavailable: true, to the tasks with the mentioned content item.
thanks!

**Comment by rundssoar** [comment]
@merit-maita I've added the "Skip if unavailable" to the two failing commands.
For the "AppendIfNotEmpty" error, i'm surprised, that this wasn't an issue in the previous release, as i didn't change that Task as part of this PR, but the Test should no longer fail.

**Review by merit-maita — APPROVED** [review]

**Comment by merit-maita** [comment]
@rundssoar im sorry but as you can see we have one last issue with the pr conflict, a new release was merged already to master with the same version as yours, so please pull from master and create new rn for your changes.
thanks!

**Comment by rundssoar** [comment]
@merit-maita I think, there is no need to update this PR, since [#33235](https://github.com/demisto/content/pull/33235) includes my changes, but adds even additional features, so i recommend merging that PR, rather than this one and we can close this PR
Thank you for your work anyway.

**Comment by merit-maita** [comment]
> @merit-maita I think, there is no need to update this PR, since [#33235](https://github.com/demisto/content/pull/33235) includes my changes, but adds even additional features, so i recommend merging that PR, rather than this one and we can close this PR Thank you for your work anyway.

i see, okay no problem i'll close this pr, thank you for your effort anyway.

**Pull request closed without merging**